compiler-error ESPFlashCounter.h:49:7: error: 'SPIFFS'

Hi
I tried to compile the Powercycle-counter with ESP32-core 1.0.6
but get this error
ESPFlashCounter.h:49:7: error: 'SPIFFS' was not declared in this scope

   if(!SPIFFS.exists(fileName)) 

I'm not sure. Seems to be an inconsistency some files using SPIFF some others FS

Is it possible to create a version 1.0.1 which makes consequent use of FS.h ?

--- a/src/ESPFlash.h
+++ b/src/ESPFlash.h
@@ -24,6 +24,10 @@
 
 #include <FS.h>
 
+#if defined(ARDUINO_ARCH_ESP32)
+#include <SPIFFS.h>
+#endif
+
 template<class T>
 class ESPFlash
 {  

